### PR TITLE
[#3765] improvement(client-python): Fixed missing-timeout Pylint rule for client-python

### PR DESCRIPTION
**Pull request by liuchunhao**
### What changes were proposed in this pull request?

This PR fixes missing-timeout Pylint rule for client-python 

- modified:
  * clients/client-python/tests/integration/integration_test_env.py    (added a reasonable timeout for requests.get)
  * clients/client-python/tests/integration/auth/test_oauth2_client.py   (added a reasonable timeout for requests.get)
  * clients/client-python/pylintrc   (removed missing-timeout from disable section in pylintrc)

- new file:
  * clients/client-python/tests/integration/config.py (added a file to store constant values)

### Why are the changes needed?

Define a reasonable timeout for requests.get to prevent program from hanging indefinitely

Fix: #3765

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?
```bash
./gradlew clients:client-python:pylint
```

### CODE REVIEW

File: clients/client-python/tests/integration/config.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Config:
```
Comment (jerryshao) [inline clients/client-python/tests/integration/config.py:1]: I think you miss the license header for this file. Please add a license header like other files.

**Comment by jerryshao** [comment]
@liuchunhao thanks a lot for your contribution.

@noidname01 would you please help to review when you have time.

File: clients/client-python/tests/integration/config.py (hunk 1 above)
Comment (liuchunhao) [reply clients/client-python/tests/integration/config.py:19]: Thank you for the heads-up. I have fixed the licensing issue.

**Comment by jerryshao** [comment]
Overall LGTM, thanks @liuchunhao for your contributions.

**Review by noidname01 — APPROVED** [review]

**Review by xunliu — APPROVED** [review]
LGTM

**Pull request merged**